.evergreen/download-mongodb.sh

```diff
@@ -401,7 +401,7 @@ get_mongodb_download_url_for ()
       ;;
       cygwin*-x86_64)
          EXTRACT="/cygdrive/c/Progra~2/7-Zip/7z.exe x"
-         MONGODB_LATEST="http://downloads.10gen.com/win32/mongodb-win32-x86_64-enterprise-windows-64-latest.zip"
+         MONGODB_LATEST="http://downloads.10gen.com/windows/mongodb-windows-x86_64-enterprise-latest.zip"
              MONGODB_42="http://downloads.10gen.com/win32/mongodb-win32-x86_64-enterprise-windows-64-${VERSION_42}.zip"
              MONGODB_40="http://downloads.10gen.com/win32/mongodb-win32-x86_64-enterprise-windows-64-${VERSION_40}.zip"
              MONGODB_36="http://downloads.10gen.com/win32/mongodb-win32-x86_64-enterprise-windows-64-${VERSION_36}.zip"
```

Comment:
Can you remove the MONGODB_LATEST links for  "cygwin*-i686)" and "windows32*)"? 
https://fastdl.mongodb.org/win32/mongodb-win32-i386-latest.zip is MongoDB 3.3.3

Can you also make all the versions in `windows64*)` match `cygwin*-x86_64)`? I think those are the same platform.